### DISCUSSION TITLE
[@types/chrome] Missing attributes for chrome.browsingData

### DISCUSSION THREAD

**Discussion by RonenISRb** [DefinitelyTyped/DefinitelyTyped#58384] [Issues with a @types package]
Hello,
according to the [documentation](https://developer.chrome.com/docs/extensions/reference/browsingData/#type-RemovalOptions
), many parts are missing.
In example:
1) For 'RemovalOptions' interface, 'excludeOrigins' and 'origins' are missing.
2) There's no definition of 'removeServiceWorkers'.

Maybe it's in purpose since they are supported from version 74 only?
If so, how can I overcome it?
I need to extend the 'RemovalOptions' interface by myself for example?

[a comment by an automated account is not shown]

**Comment by nicolas377** [comment]
Hey @RonenISRb! Thanks for reporting this! This isn't purposeful, we support many API's since Chrome 74. I'll try and get right on that issue today with a PR. For now, you can extend the defs through another .d.ts file. Since the PR system is fairly quick, I'd recommend just using an `@ts-ignore` comment until the PR gets merged.

**Reply by nicolas377** [reply]
PR is up! See #58391.

**Reply by RonenISRb** [reply]
Amazing, waiting for the update :) 
it will be in version 0.0.177 of @types/chrome yeah ?

**Reply by nicolas377** [reply]
Yep! It just got merged, so the update should be up in a few minutes.
